ui/server/src/main/scala/pl/touk/nussknacker/ui/process/ProcessObjectsFinder.scala

```diff
@@ -59,6 +60,23 @@ object ProcessObjectsFinder {
       .mapValues(_.size)
   }
 
+  def findComponentProcess(processes: List[ProcessDetails], componentId: String): List[(String, ProcessDetails)] =
```

Comment:
I wonder, can we express this method in terms of the method before? E.g. somehow merge the logic to obtain Map[ComponentId, ProcessDetails], and then do either get on map, or extract id from values

Reply:
Done